draft-privacypass-rate-limit-tokens.md

```diff
@@ -306,9 +306,12 @@ and are located at the well-known location /.well-known/token-issuer-directory.
 Clients receive challenges for tokens, as described in {{AUTHSCHEME}}.
 
 For the rate-limited token issuance protocol described in this document,
-the token challenge MUST be interactive and per-origin. That is, the
-TokenChallenge structure MUST contain both the redemption_nonce and
-origin_name fields.
+the name of the origin is sent in an encrypted message from the Client
+to the Issuer. If the TokenChallenge.origin_info field contains a single
+origin name, that origin name is used. If the origin_info field contains
+multiple origin names, the client selects the single origin name that
+presented the challenge. If the origin_info field is empty, the
+encrypted message uses a special value, defined in {{encrypt-origin}}.
```

Comment:
```suggestion
encrypted message is the empty string "".
```
(We also use the special value "" in #request-two, so I figure it's simplest to just inline here.)